components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -283,6 +287,38 @@ func parseLocationSpecFromCLIArgs(location string) (backup.LocationSpecification
 			SecretKey:    backupCmdFlags.s3SecretKey,
 			SessionToken: backupCmdFlags.s3SessionToken,
 		}, nil
+	} else if strings.HasPrefix(location, "gcs://") {
```

Comment:
we should also accept "gs://"

Reply:
I believe we should use gs as it's what is what is provided by the google ui for GCS as well as what is supported in their CLI for storage management [](https://cloud.google.com/storage/docs/gsutil)